app/controllers/v0/vic/profile_photo_attachments_controller.rb

```diff
@@ -18,8 +21,13 @@ def create
       end
 
       def show
-        form_attachment = ::VIC::ProfilePhotoAttachment.where(guid: params[:id]).first
-        render(json: form_attachment)
+        guid = params[:id]
+
+        raise ActiveRecord::RecordNotFound unless GUID_PATTERN.match(guid)
+
+        form_attachment = ::VIC::ProfilePhotoAttachment.where(guid: guid).first
```

Comment:
Does this schema have an index on the guid column, and if not, do we think one is warranted?

Reply:
All it has at the moment is this: https://github.com/department-of-veterans-affairs/vets-api/blob/master/db/schema.rb#L85

`add_index "form_attachments", ["guid", "type"], name: "index_form_attachments_on_guid_and_type", unique: true, using: :btree
`

Reply:
That should work since we're using single table inheritance right - so any query against ProfilePhotoAttachment is going to implicitly have a `where type = blah` on it.

Reply:
Good point. That should be enough then!